packages/api/cms-api/src/document/__tests__/validateNotModified.test.ts

```diff
@@ -0,0 +1,39 @@
+import { validateNotModified } from "../validateNotModified";
+
+describe("validateNotModified", () => {
+    it("should throw an error if updateAt from document is invalid ", async () => {
+        expect(() => {
+            validateNotModified(
+                {
+                    id: "1234",
+                    updatedAt: new Date("invalid Date"),
+                },
+                new Date(),
+            );
+        }).toThrowError("Conflict: Document has been modified.");
+    });
```

Comment:
IMO this should throw an error that the provided updatedAt is invalid, not a conflict error

Reply:
agree, but then i would need to change the implemented logic, i basically just wanted to test if the basic behaviour did not change.

Reply:
I'd remove this test and rather add one for the first if: https://github.com/vivid-planet/comet/blob/renovate/date-fns-4.x/packages/api/cms-api/src/document/validateNotModified.ts#L6-L8

Reply:
updated test.